backend/layers/processing/process_download_validate.py

```diff
@@ -154,11 +154,16 @@ def _get_batch_condition() -> Optional[str]:
             mean_genes_per_cell=numerator / denominator,
             is_primary_data=_get_is_primary_data(),
             cell_type=_get_term_pairs("cell_type"),
-            x_approximate_distribution=_get_x_approximate_distribution(),  # TODO: pay attention
```

Comment:
🙅 what a helpful TODO message?

Reply:
I know LOL

I have no idea what this is referring to, so I guess I was not paying sufficient attention